dandi-info/config.yaml.j2

```diff
@@ -61,10 +60,6 @@ hub:
           )
           c.CryptKeeper.keys = [ os.urandom(32) ]
```

Comment:
I think this section that is checking for the JUPYTERHUB_CRYPT_KEY could be problematic. It may actually be set without the environment variable in the latest version of z2jh.

I suggest removing this section. In the latest version of Z2JH, these are automatically generated for you as well if they aren't explicitly set.

Reply:
if you know of a quick pointer to fix the crypt key issue that would be great:

```
$ helm upgrade -f config.yaml dandihub-jupyterhub jupyterhub/jupyterhub --timeout 3600


hub pod crashes and the log says.

auth_state is enabled, but encryption is not available: Encryption keys must be specified in JUPYTERHUB_CRYPT_KEY env
```

i set this manually:

```
export JUPYTERHUB_CRYPT_KEY=$(openssl rand -hex 32)
```

but it still crashes:

`hub-xxxxx              0/1     CrashLoopBackOff   3          79s`

seems to not even restart the hub.  (i'm using chart 0.11.1 release) should i upgrade to the version you pointed to?

Reply:
trying dev chart release now

Reply:
that doesn't work unless i move to helm 3 i guess. this may require full shutdown of the hub.